docs/docs/dashboard.md

```diff
@@ -76,7 +101,8 @@ When the app(JSON file) that includes the table schema is imported, and the tabl
 
 If the table with the same name is already present in the workspace, the new table will be created with the name `<table name>_<unix timestamp>`. Example: `<tablename>_1627980000`.
```

Comment:
what is this for?

Reply:
Ignore this section. That's not the part of readable url.